### PR TITLE
translate eslint/prettier piece

### PR DESCRIPTION
**Pull request by georgem3**

### CODE REVIEW

File: sections/codestylepractices/eslint_prettier.german.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+# Verwendung von ESLint und Prettier
+
+
+### ESLint und Prettier im Vergleich
+
+Wenn Du diesen Code mit ESLint formatierst, erhältst Du lediglich eine Warnung, dass er zu breit ist (abhängig von der `max-len` Einstellung). Prettier formatiert es automatisch für Dich.
```
Comment (BrunoScheufler) [inline sections/codestylepractices/eslint_prettier.german.md]: Maybe I'd use a lowercase "Du" here, although both forms are possible I'd prefer the lowercased standard

Comment (georgem3) [reply sections/codestylepractices/eslint_prettier.german.md]: Will do! Will probably fix it in a second commit, along with another translated piece

**Comment by BrunoScheufler** [comment]
@georgem3 second piece looks great as well! Once you’re done, simply write some message to indicate that, I'll review one final time and merge then 👍

**Comment by georgem3** [comment]
message, as in comment? :D the PR is done from my side 😄

**Comment by i0natan** [comment]
@BrunoScheufler Shall we add @georgem3 to July maintenance?

**Pull request merged**